analysis_driver/notification.py

```diff
@@ -1,4 +1,59 @@
-from egcg_core import notifications
+from egcg_core import notifications, clarity
+from pyclarity_lims.entities import Step
+from analysis_driver.exceptions import PipelineError
+
+
+class LimsNotification():
+    def __init__(self, name):
+        self.name = name
+        self.sample_list = clarity.get_list_of_samples(self.name)
```

Comment:
We should only retrieve one and only one sample here.
`clarity.get_sample()` would be more appropriate and we should check the return is not `None`